packages/mjml-head-breakpoint/README.md

```diff
@@ -1,5 +1,5 @@
 ## mj-breakpoint
-This tag allows you to control on which breakpoint the layout should go desktop/mobile.
+This tag allows you to control on which breakpoint the layout should go desktop/mobile. At window widths below this value the layout will be displayed in mobile mode.
```

Comment:
I'm not sure we should add this in the doc here as everyone should be familiar enough with media queries and what a breakpoint mean. Can you remove this ?

Reply:
Sure, if it's too verbose.